libs/common/files.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include "logger/logger.hpp"
 
-void iroha::remove_all(const std::string &dump_dir) {
+void iroha::remove_dir_contents(const std::string &dump_dir) {
```

Comment:
can be removed in favor of boost::remove_all

Reply:
It doesn't remove the base folder, thus can't